Class-18-(ORM-Part-3).md

```diff
@@ -0,0 +1,26 @@
+# Class 18: Object-Relational Mapping Part 3
+
+## In the prep work this week, the students learned
+
+1. How to build persistent model classes with one-to-one and many-to-many relationships.
+1. What a composite key is.
+1. What a join table is.
+1. How to use different annotations such as ``@OneToOne`` and ``@ManyToMany``.
+
+## Announcements
+
+1. Assignment #4 is due soon!
```

Comment:
I would also say that with today's material, they have all they need to complete it.

Reply:
Done!